package.json

```diff
@@ -1,15 +1,73 @@
 {
```

Comment:
You've committed a npm checked out package.json instead of source package.json.

Reply:
Fixes: https://github.com/Elyx0/react-native-document-picker/pull/80/commits/9f2ae818b44bba8652caa570a5b48fe6cedae991